app/util/sentry/utils.test.ts

```
@@ -126,4 +128,360 @@ describe('captureSentryFeedback', () => {
       expectedUserFeedback,
     );
   });
+
+  describe('maskObject', () => {
+    const rootState = {
```

Q: how to make sure this will be in sync with our real state? it will have to be updated each time the real rootState will be updated in the app.
Should we move this to TS to be able to implement RootState from app/reducer and ensure a link between both forcing des to update this mask test too?

---

Not sure if I understood this part "Should we move this to TS to be able to implement RootState"
But rootState it's meant to be a default state, default state doesn't have data, we want here to see if the mask is working as expected and we need to mock data for that to happen!